packages/palette/src/elements/Select/Select.tsx

```diff
@@ -159,3 +190,34 @@ const SmallSelectContainer = styled.div<SelectProps>`
 
   ${styledSpace};
 `
+
+const MiniSelectContainer = styled.div<SelectProps>`
+  position: relative;
+
+  select {
+    ${hideDefaultSkin};
+    background-color: ${color("black10")};
+    border-radius: 2px;
+    font-size: ${themeGet("typeSizes.sans.2.fontSize")}px;
+    line-height: ${themeGet("typeSizes.sans.2.lineHeight")}px;
+    padding: ${space(0.5)}px ${space(1) + carretSize * 4}px ${space(0.5)}px
+      ${space(1)}px;
+
+    &:hover {
+      background-color: ${color("black30")};
+    }
+    &:focus {
+      border-color: ${color("purple100")};
+    }
+  }
+
+  &::after {
+    ${caretArrow};
+    content: "";
+    cursor: pointer;
+    margin-left: -${space(1) + carretSize * 2}px;
+    pointer-events: none;
+    position: absolute;
+    top: ${5 + space(0.5)}px;
```

Comment:
This 5 is a bit of a magic number here. I basically positioned the carret properly without any padding and that is what 5 came from and then added the padding. Not sure what is the more logical way to get to it here.

Reply:
I believe 
```js
5 + space(0.5) === space(1)
```

Reply:
Absolutely true, but the idea here was that it's some magical number 5 + whatever is currently set as top padding for this element. But since the padding is just hardcoded above - maybe that is an unnecessary complication here.